app/components/provider_interface/provider_user_details_component.rb

```diff
@@ -14,22 +14,44 @@ def rows
       [
         { key: 'Name', value: @provider_user.full_name },
         { key: 'Email', value: @provider_user.email_address },
-        permissions_row,
-      ]
+      ] + permission_rows
     end
 
-    def permissions_row
-      {
-        key: 'Permissions',
-        value: render(
-          ProviderInterface::ProviderPermissionsListComponent.new(
-            provider_permissions: @provider_permissions,
-            possible_permissions: @possible_permissions,
-          ),
-        ),
-        change_path: provider_interface_provider_user_edit_providers_path(@provider_user),
-        action: 'Change',
-      }
+    def visible_provider_permissions
+      @possible_permissions & @provider_permissions
+    end
+
+    def permission_rows
+      visible_provider_permissions.map do |permission|
+        {
+          key: "Permissions: #{permission.provider.name}",
+          value: temp_code_this_will_be_a_component(permission),
```

Comment:
This is where your component will be included/rendered @AgaDufrat :)

Reply:
I made some changes to Tijmen's component in the end so we can go with:
```suggestion
          value: render(PermissionsList.new(permission)),
```